packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -20,23 +20,41 @@ import { plugin as requestAuthPlugin } from 'mwp-auth-plugin';
 const CSRF_COOKIE_NAME = 'x-mwp-csrf';
 const CSRF_HEADER_COOKIE_NAME = `${CSRF_COOKIE_NAME}-header`;
 const CSRF_HEADER_NAME = CSRF_COOKIE_NAME;
+// dev cookie names
+const DEV_CSRF_COOKIE_NAME = `${CSRF_COOKIE_NAME}-dev`;
+const DEV_CSRF_HEADER_COOKIE_NAME = `${DEV_CSRF_COOKIE_NAME}-header`;
```

Comment:
Thinking about this a bit more - why not just keep all the implementation logic the same, and just make the initial assignment of `CSRF_COOKIE_NAME` and `CSRF_HEADER_COOKIE_NAME` match the environment? i.e.

```js
const CSRF_COOKIE_NAME = process.env.NODE_ENV === 'production'
  ? 'x-mwp-csrf'
  : 'x-mwp-csrf_DEV';
const CSRF_HEADER_COOKIE_NAME = process.env.NODE_ENV === 'production'
  ? 'x-mwp-csrf-header'
  : 'x-mwp-csrf-header_DEV';
```

I think if you set up the values that way, you don't have to do any more juggling of values in the rest of the module.